scripts/copy-backends.py

```
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+"""
+Cross-platform equivalent of:
+  find ./target/ -type f -executable -name 'pixi-build*' -exec cp {} $PREFIX/bin \;
+
+- Recursively scans ./target for files named 'pixi-build*'
+- Keeps only executables (POSIX: x bit; Windows: PATHEXT)
+- Copies matches into $PREFIX/bin
+"""
+from pathlib import Path
+import os
+import sys
+import shutil
+
+def is_executable(path: Path) -> bool:
+    if not path.is_file():
+        return False
+    if os.name == "nt":
+        pathext = os.environ.get("PATHEXT", ".EXE;.BAT;.CMD;.COM;.PS1").lower().split(";")
+        return path.suffix.lower() in pathext
+    return os.access(str(path), os.X_OK)
+
+def main() -> int:
```

Let's raise exceptions instead of returning integers